examples/wave_example.ipynb

```diff
@@ -25,154 +25,1751 @@
+    "                                                        # Convert to pandas Series \n",
     "\n",
-    "# Generate bins for Hm0 and Te\n",
-    "Hm0_bins = np.arange(0, Hm0.max() + .5, .5) # input is start, stop, and step size\n",
-    "Te_bins = np.arange(0, Te.max() + 1, 1)\n",
-    "\n",
-    "# Create capture length matrices using mean, standard deviation, count, min and max statistics\n",
-    "LM_mean = wave.performance.capture_length_matrix(Hm0, Te, L, 'mean', Hm0_bins, Te_bins)\n",
-    "LM_std = wave.performance.capture_length_matrix(Hm0, Te, L, 'std', Hm0_bins, Te_bins)\n",
-    "LM_count = wave.performance.capture_length_matrix(Hm0, Te, L, 'count', Hm0_bins, Te_bins)\n",
-    "LM_min = wave.performance.capture_length_matrix(Hm0, Te, L, 'min', Hm0_bins, Te_bins)\n",
-    "LM_max = wave.performance.capture_length_matrix(Hm0, Te, L, 'max', Hm0_bins, Te_bins)\n",
-    "\n",
-    "# Print capture length matrices using mean\n",
-    "print(LM_mean)"
+    "Hm0 = wave.resource.significant_wave_height(ndbc_data).squeeze() # Compute significant wave height from spectra and \n",
+    "                                                                 # Convert to pandas Series \n",
+    "print(J)\n",
+    "print(Hm0)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Additional capture length matrices can be computed, for example, the frequency matrix is computed below."
+    "## Generate Random Power Data\n",
+    "\n",
+    "For demonstration purposes, this example uses synthetic power data generated from statistical distributions.  In a real application, the user would provide power values from a WEC.  The data is stored in pandas Series, containing 743 points. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 7,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Create capture length matrices using frequency\n",
-    "LM_freq = wave.performance.capture_length_matrix(Hm0, Te, L,'frequency', Hm0_bins, Te_bins)"
+    "np.random.seed(1)                                               # Set the random seed, to reproduce results\n",
+    "P = pd.Series(np.random.normal(200, 40, 743),index = J.index)   # Generate random power values"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Power Matrices\n",
-    "As specified in IEC/TS 62600-100, the power matrix is generated from the capture length matrix and wave energy flux matrix, as shown below"
+    "## Capture Length Matrices\n",
+    "\n",
+    "The following operations create capture length matrices, as specified by the IEC/TS 62600-100.  But first, we need to calculate capture length and define bin centers.  The mean capture length matrix is printed below.  Keep in mind that this data has been artificially generated, so it may not be representative of what a real-world scatter diagram would look like."
```

Comment:
@kmruehl is this section you said was removed?

Reply:
@ssolson yes. I think the issue is related to GitHub rendering of diffs on `*.ipynb` files. The best option is to view the files locally, but when working off of GitHub I've found that the best way to avoid my mistake is to "view changes", "view file", and choose "display the rendered blob", as opposed to viewing the diff.